App.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View,Image } from 'react-native';
 
 export default class App extends React.Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text>Open up App.js to start working on your app!</Text>
+        <Text style={styles.bigShopUHeader}> Let's shopU! </Text>
```

Comment:
There's a lot of red that's popping up. The reason for this is that the apostrophe is considered a special character; you have to escape the character so it becomes a normal character.

Instead of just typing ' you should `{ `'` }`